services/galley/src/Galley/Data/CustomInstance.hs

```diff
@@ -0,0 +1,23 @@
+{-# LANGUAGE RecordWildCards #-}
+module Galley.Data.CustomInstance
+    ( getCustomInstance
+    , setCustomInstance
+    ) where
+
+import Imports
+import Cassandra
+import Galley.Data.Instances ()
+import Galley.Types
+
+import qualified Galley.Data.Queries  as Cql
+
+getCustomInstance :: MonadClient m => EmailDomain -> m (Maybe CustomInstance)
+getCustomInstance domain = fmap toCustomInstance <$> do
+    retry x1 $ query1 Cql.selectCustomInstance (params Quorum (Identity domain))
+  where
+    toCustomInstance (customInstanceConfigJsonUrl, customInstanceWebappWelcomeUrl, customInstanceBlockCloudUsers) =
+        CustomInstance{..}
+
+setCustomInstance :: MonadClient m => EmailDomain -> CustomInstance -> m ()
+setCustomInstance domain CustomInstance{..} = do
+    retry x5 $ write Cql.updateCustomInstance (params Quorum (customInstanceConfigJsonUrl, customInstanceWebappWelcomeUrl, customInstanceBlockCloudUsers, domain))
```

Comment:
Why is `domain` the last argument here?  In the schema it looks like it should be the first?

(ah, i see, it just has to be consistent with `Cql.updateCustomInstance`.  still, would be nice to have it consistent all over.  anyway, never mind.)